.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: All the CI
+
+on: [push, pull_request]
+
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Unit Tests
+      run: |
+        npm ci
+        npm run test
```

Comment:
Thinking of dropping this. Github has 10.x already installed, so I don't think we need to worry about specifying a node version. This step took a long time in the actions run

_multiline comments!_

Reply:
For reference, https://help.github.com/en/articles/software-in-virtual-environments-for-github-actions#ubuntu-1804-lts : Node.js (v10.16.3)

Reply:
Right, and I don't think we're going to run into version issues... ever